pyproject.toml

```diff
@@ -138,6 +138,7 @@ disable_error_code = ['import-not-found', # Unable to locate module or package s
                 'no-untyped-call', # Function calls are only made to functions that are fully typed
                 'type-arg', # Requires type arguments to be specified in list[] and dict[]
                 'union-attr', # Item "float" of "float | Any" has no attribute "reshape"
+                'warn-return-any', # Returning Any from a function
```

Comment:
@subagonsouth this is to avoid that weird issue with typing in hi_l1c where I think you are returning the correct type and I can get it inferred locally, but mypy doesn't like it. This seems inconsequential so lets just skip over it IMO.